app/packages/looker-3d/src/renderables/pointcloud/index.tsx

```diff
@@ -72,20 +74,21 @@ export const PointCloudMesh = ({
     }
   }, [boundingBox, pointsGeometry, points, onLoad]);
 
-  if (minZ === undefined) {
+  if (!minZ) {
```

Comment:
Ah, good point! I should have checked for `null` + `undefined` since that's what was causing the issue:
```
  const minZ = _.get(settings, "pointCloud.minZ", null);
```